charts/hpe-csi-driver/v1.4.2/questions.yml

```diff
@@ -0,0 +1,39 @@
+labels:
+  io.rancher.certified: partner
+categories:
+- storage
+questions:
+- variable: imagePullPolicy
+  label: "ImagePullPolicy"
+  default: "IfNotPresent"
+  type: enum
+  options:
+    - "IfNotPresent"
+    - "Always"
+    - "Never"
+  description: "ImagePullPolicy for all CSI driver images"
+  group: "HPE CSI Driver settings"
+- variable: disableNodeConformance
+  label: "Disable automatic installation of iSCSI/Multipath Packages"
+  type: boolean
+  default: false
+  description: "Disable automatic installation of iSCSI/Multipath Packages"
+  group: "HPE CSI Driver settings"
+- variable: iscsi.chapUser
+  label: "iSCSI CHAP Username"
+  type: string
+  required: false
+  description: "Specify username for iSCSI CHAP authentication"
+  group: "HPE iSCSI settings"
+- variable: iscsi.chapPassword
+  label: "iSCSI CHAP Password"
+  type: password
+  required: false
+  description: "Specify password for iSCSI CHAP authentication"
+  group: "HPE iSCSI settings"
+- variable: registryPrefix
+  label: "registry"
```

Comment:
Nit: Please capitalize registry. This is what's displayed in the UI and is the only one in lowercase currently.

Reply:
Thanks for catching this. There was another issue that had snuck in there. I'm correcting this in our upstream chart too.